tools/gce-quota-sync/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.8
+
+ENV CRONSPEC "0 * * * *"
```

Comment:
One last thought - if we're providing cron defaults, we should avoid 'crowded' slots like top of the hour. Throw a favourite prime number from 1 to 59 in the minute column.